run-tests.sh

```diff
@@ -260,6 +260,9 @@ runTestsIfServiceVersion "@requestor" "requestor" "1.5.0" "2022.02"
 runTestsIfServiceVersion "@requestor" "arborist" "3.2.0" "2021.12"
 runTestsIfServiceVersion "@requestorNew" "requestor" "1.5.1" "2022.06"
 runTestsIfServiceVersion "@requestorNew" "arborist" "3.2.0" "2021.12"
+runTestsIfServiceVersion "@clientCreds" "arborist" "2022.10"
+runTestsIfServiceVersion "@clientCreds" "fence" "6.1.0" "2022.10"
+runTestsIfServiceVersion "@clientCreds" "requestor" "2022.10"
```

Comment:
reminder: before merging the arborist and requestor lines will need to be updated here once the services are updated

Reply:
👍